modules/dnn/test/test_int8_layers.cpp

```diff
@@ -1269,6 +1308,9 @@ TEST_P(Test_Int8_nets, YOLOv4_tiny)
     {
         SCOPED_TRACE("batch size 1");
         testDarknetModel(config_file, weights_file, ref.rowRange(0, N0), scoreDiff, iouDiff, confThreshold);
+
+        // per-tensor quantize
+        testDarknetModel(config_file, weights_file, ref.rowRange(0, N0), scoreDiff, 0.16, 0.7, 0.4, true);
```

Comment:
BTW, It makes sense to create dedicated test or at least use `SCOPED_TRACE();`.
(here and above)

Reply:
Ok, I will try to update it.